native/src/modules/layout/components/Header.js

```diff
@@ -40,8 +40,6 @@ const Icon = styled.Image`
 const HeaderText: StyledComponent<{||}, ThemeType, *> = styled.Text`
   flex: 1;
   flex-direction: column;
-  text-align-vertical: center;
```

Comment:
Shouldn't this actually center it vertically? I trust you if you say yes :D

Reply:
It did not, at least not on iOS apparently :D I just removed the height and let the parent container do the work

Reply:
But continues to work on android?

Reply:
That is the plan, yes. @Taggotty and @SteffiStuffel already tested but I will test again myself to make sure.

Reply:
Yes it does :)